### DISCUSSION TITLE
`build_op_context` doesn't seem to support `context.partition_time_window`

### DISCUSSION THREAD

**Discussion by chrishiste** [dagster-io/dagster#18714] [Q&A] [answered]
Hello, I couldn't find a way to mock `context.partition_time_window` of an op using `build_op_context` is this not supported at the moment?

Thanks in advance

**Comment by sryza** [comment, marked as answer]
It looks like this is currently missing functionality. It's track here, where a workaround is also listed: https://github.com/dagster-io/dagster/issues/13557.
